samples/suit/smp_transfer/README.rst

```diff
@@ -0,0 +1,392 @@
+
+    * ``dfu_partition`` is where the update candidate is stored before the update process begins.
+
+Manifest template files
+=======================
+
+All required manifest template files are automatically created after the sample is built for the first time.
+These are needed to create a SUIT envelope to carry out the DFU.
+
+.. note::
+
+   The Radio Domain manifest template is only created when building the Bluetooth Low Energy version of the sample, and not the UART version.
+   Currently, it is not needed for the UART version.
+
+If you want to make modifications to how the DFU is executed in this sample, you can do so by editing the manifest templates.
+See the :ref:`ug_nrf54h20_suit_customize_dfu` user guide for instructions and examples.
+
+Building and running
+********************
+
+.. |sample path| replace:: :file:`samples/suit/smp_transfer`
+
+This sample can be found under |sample path| in the |NCS| folder structure.
+
+Building and programming using the command line
+===============================================
+
+To build and program the sample to the nRF54H20 DK, complete the following steps:
+
+.. tabs::
+
+   .. group-tab:: Over Bluetooth Low Energy
+
+      1. Open a terminal window in |sample path|.
+      #. Build the sample using the following ``west`` command, with the following Kconfig options set:
+
+         .. code-block:: console
+
+            west build -p -b nrf54h20dk/nrf54h20/cpuapp --sysbuild -- -DFILE_SUFFIX=bt -DCONFIG_SUIT_ENVELOPE_SEQUENCE_NUM=1
+
+         .. note::
+
+            If you are compiling in Windows and the build is unsuccessful due to the maximum path length limitation, use the following command:
+
+            .. code-block:: console
+
+               west build -p -b nrf54h20dk/nrf54h20/cpuapp -d C:/ncs-lcs/work-dir --sysbuild -- -DFILE_SUFFIX=bt -DCONFIG_SUIT_ENVELOPE_SEQUENCE_NUM=1
+
+         If you want to further configure your sample, see :ref:`configure_application` for additional information.
+
+         After running the ``west build`` command, the output build files can be found in the :file:`build/DFU` directory.
+         The following build artifacts are found with both firmware binaries embedded as integrated payloads:
+
+          * :file:`root.suit` - This file is the most important envelope, containing the embedded :file:`application.suit` and :file:`radio.suit` files as integrated dependencies.
+            These files are used to execute DFU.
+          * :file:`application.suit` - This file is the envelope for the Application Core, containing embedded Application Core firmware as an integrated payload.
+          * :file:`radio.suit` - This file is the envelope for the Radio Core, containing embedded radio core firmware as an integrated payload.
+
+         For more information about files generated as output of the build process, see :ref:`app_build_output_files`.
```

Comment:
Can we have these files added to the list in :ref:`app_build_output_files` too? You could for example add a section there for SUIT output files, or even a whole section for DFU output files with SUIT as a subsection of this new section?

Reply:
Will do

Reply:
Added (will be reflected in the next push). 
Also made a ticket to review the Output files page again for new SUIT section before 2.7, as need SMEs to confirm if list contains all output files, or if names have changed.